src/backend/users/views.py

```diff
@@ -147,6 +147,17 @@ def get(self, request, **kwargs):
 
         return Response(content)
 
+    def process_nullable_int(self, data, field):
+        try:
+            result = int(data[field])
+        except ValueError:
```

Comment:
```suggestion
        except (ValueError, TypeError):
```

Catching the `ValueError` helped part of the time for me, but anytime I tried to add `nonVoucherBudget` but not both `voucherRooms` and `nonVoucherRooms` or vice versa, it failed with a `TypeError` and the message "int() argument must be a string, a bytes-like object or a real number, not 'NoneType'".

Reply:
Ah good call. Thanks!